scripts/proto.lock

```diff
@@ -10263,6 +10268,11 @@
                 "name": "logical_table_id",
                 "type": "int64"
               },
+              {
+                "id": 7,
+                "name": "api_version",
```

Comment:
why hasn't it been updated before?

Reply:
I assume that the previous guy didn't submit the `proto.lock` file 🤔.

Reply:
@Connor1996 can u take a look at https://github.com/pingcap/kvproto/pull/1058? :)